rpc/core/bridge_commitments.go

```diff
@@ -0,0 +1,234 @@
+package core
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strconv"
+
+	"github.com/cometbft/cometbft/crypto/merkle"
+	ctypes "github.com/cometbft/cometbft/rpc/core/types"
+	rpctypes "github.com/cometbft/cometbft/rpc/jsonrpc/types"
+	"github.com/cometbft/cometbft/types"
+)
+
+const (
+	// BridgeCommitmentBlocksLimit is the limit to the number of blocks we can generate a bridge commitment for.
+	// This limits the ZK prover time required to compute.
+	BridgeCommitmentBlocksLimit = 1000
```

Comment:
How did we come up with 1000 specifically?

Reply:
celestia

Reply:
Link

Reply:
Added